### ISSUE TITLE
Fix VOMemoryRepositoryTest

### ISSUE THREAD

**Issue by tinchodias** [pharo-nosql/voyage#72]
2 failures and 8 errors out of 16 tests!

**Comment by noha** [comment]
I don't have a class VOMemoryRepositoryTest. Where is it?

**Comment by tinchodias** [comment]
Ah, I forgot it's not isntalled by default! It's in 'memory tests' group in BaselineOfVoyage.

**Comment by estebanlm** [comment]
is this still an issue?

**Comment by tinchodias** [comment]
Yes, I just reproduced it: 
- download pharo 61+vm 
- eval:
```
Metacello new 
	repository: 'github://pharo-nosql/voyage/mc';
	baseline: 'Voyage';
	load: #('mongo tests' 'memory tests').
```
- run tests: 8 errors, 2 failures

**Comment by tinchodias** [comment]
Is there some good reason to keep the memory repository? 

I propose to remove it and, if needed, recover from git history in the future.

**Comment by noha** [comment]
Why you want to remove it? It is important at least for writing tests

**Comment by tinchodias** [comment]
Two reasons:
- I see there is the `aBlock asMongoQuery` to transform to the mongodb format dictionary, but the VOMemoryRepository would need the inverse transformation, which is not implemented... or did I miss it?
- I didn't check since when it is broken but it looks like if nobody cares. 

But I have nothing against fixing and keeping it... Just tried to be pragmatic.

**Comment by tinchodias** [comment]
Ok... I've read #48 and now I realize that VOMemoryRepository shouldn't be prepared to receive mongo queries as arguments as it's not a subclass of VOMongoRepository: just block closures.

**Comment by noha** [comment]
Is this resolved?

**Comment by tinchodias** [comment]
current master in pharo 6 gives me I get 2 fails and 8 errors in 'memory tests'

**Comment by noha** [comment]
will be fixed in #120

**Comment by noha** [comment]
This is fixed. The test with the mongo query has been moved to mongo. Memory tests work again
